std/range/package.d

```diff
@@ -2465,11 +2466,13 @@ auto takeOne(R)(R source) if (isInputRange!R)
             void popFront()
             {
                 assert(!empty, "Attempting to popFront an empty takeOne");
+                _source.popFront();
                 _empty = true;
             }
             void popBack()
             {
                 assert(!empty, "Attempting to popBack an empty takeOne");
+                _source.popFront();
```

Comment:
I aliased popBack to popFront. They do the same thing so this perhaps removes some bloat.